### PR TITLE
Completing the QRNG tutorial

### PR DESCRIPTION
**Pull request by geduardo**
Final PR for the update discussed in the draft PR #638. It also fixes the issue #648. 
Main things:
- Completes the QRNG tutorial with the sample code to create a random integer generator. 
- Changes the location of the image files and the format to avoid transparencies in the dark mode.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f958616314fcd29ff9650351ad4df62cf02a44ca/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8eb41e76-ebeb-4db4-a87e-f3a75615aa02_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f958616314fcd29ff9650351ad4df62cf02a44ca/articles/quickstarts/qrng.md) | 79 | 63 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/59252e6e-e487-4077-bc19-d643e1686982_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0471897849c66c096309e5140c439ca160554706/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b28fa5cd-20ce-425a-a090-cf1c4ef6f8b8_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0471897849c66c096309e5140c439ca160554706/articles/quickstarts/qrng.md) | 79 | 63 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5a0903d4-db75-46c8-8e6d-eea5942ebc50_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/20b97c284b8aa8de4c1b65806ebffab6baaf11d1/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3612342c-e491-4ee4-8013-60dcd5f703b9_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/20b97c284b8aa8de4c1b65806ebffab6baaf11d1/articles/quickstarts/qrng.md) | 84 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/976d57dc-f211-493a-b6a1-3abe3d21d9f8_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d97ac7bb1d76dad7a6f9334efe6559796c6bfdf/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a2fe52c0-85a1-4cca-962d-369684113bb3_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d97ac7bb1d76dad7a6f9334efe6559796c6bfdf/articles/quickstarts/qrng.md) | 86 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a03f9c7-c328-430e-8e7f-cb95af73a277_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/quickstarts/qrng.md [hunk 1]
```diff
@@ -51,18 +51,61 @@ When a `Qubit` is de-allocated it must be explicitly set back to the `Zero` stat
...
 * Then we measure the qubit and save the output:
 
-<img src="./Measurement2.svg" width="450">
+<img src="~/media/qrng-meas.png" width="450">
 
 Since the outcome of the measurement is completely random, we have obtained a random bit. We can call this operation several times to create integers. For example, if we call the operation three times to obtain three random bits, we can build random 3-bit numbers (that is, a random number between 0 and 7).
+
+## Creating a complete random number generator using a host program
+
+Now that we have a Q# operation that generates random bits, we can use it to build a complete random number generator with a host program.
```
Comment (cgranade) [inline articles/quickstarts/qrng.md]:
```suggestion
Now that we have a Q# operation that generates random bits, we can use it to build a complete quantum random number generator with a host program.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/11f99cb8d8ea1a2daa4d7ca19c9c84376ffc26cd/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2b8b9b7b-cfa9-4ce1-856d-5a0a37353130_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/11f99cb8d8ea1a2daa4d7ca19c9c84376ffc26cd/articles/quickstarts/qrng.md) | 86 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fc2f0f73-cfc3-4f70-85c7-774850bca020_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f0f0bab783bbb2b34d6a53911ed852f2400d7902/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/44374c44-6d20-43e5-8cf1-74145c207da4_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f0f0bab783bbb2b34d6a53911ed852f2400d7902/articles/quickstarts/qrng.md) | 86 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f7f36f2c-c7fe-48a6-9fcf-2dca75230ae1_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/938702d559602baaeabd8d563103301bc982397f/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9978227b-79d5-4afb-ac48-7b44ee89ee97_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/938702d559602baaeabd8d563103301bc982397f/articles/quickstarts/qrng.md) | 86 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5b80de29-9603-4b72-8954-aab74b4af0e7_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5509b0932c0b7f7f18bb5f78589487dcaded4766/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8b4f23d3-a2e7-4653-8159-030a6f9bac21_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5509b0932c0b7f7f18bb5f78589487dcaded4766/articles/quickstarts/qrng.md) | 87 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/18b1c4ad-714f-4c65-af4e-314aef8d82c7_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/488a52df6d4d8180e93807246949403a82d2fca0/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/144bdc92-e511-454c-8ab4-913cb472836e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/488a52df6d4d8180e93807246949403a82d2fca0/articles/quickstarts/qrng.md) | 87 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/680d5f34-456e-4b96-a914-63eb91321cce_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f62d2851de7a2fea042e8d824b2455d9bc91a422/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/80e5f6fd-0df6-4371-ad29-c3c97fdaebdd_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f62d2851de7a2fea042e8d824b2455d9bc91a422/articles/quickstarts/qrng.md) | 87 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1154d066-4fd4-47dc-9446-480e3bae9f89_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/959cb3d29debd3ede38e9cab9b0251cc4f9aee55/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0fc0dba-e672-44e8-9cf9-890ed7d67bb2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/959cb3d29debd3ede38e9cab9b0251cc4f9aee55/articles/quickstarts/qrng.md) | 87 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/df16d281-7f03-4236-afae-5965fb58ec80_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/search.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/959cb3d29debd3ede38e9cab9b0251cc4f9aee55/articles/quickstarts/search.md) | 80 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/481e2b91-ec6d-4c7a-90cb-002bb25f8325_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/cb276fbb5b14a7132fb946948f77255d5606a27a/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4e6768d-4aaf-43ab-8c62-eaf5379f048f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/cb276fbb5b14a7132fb946948f77255d5606a27a/articles/quickstarts/qrng.md) | 87 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f3e9a8cc-b14f-430f-b00b-e0e6de9e9731_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/search.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/cb276fbb5b14a7132fb946948f77255d5606a27a/articles/quickstarts/search.md) | 80 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f28bcdce-18ab-4256-9901-0cb21f233bb6_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b427300ea6e59b774b96a3aa248ae20c6b3b623b/articles/install-guide/pyinstall.md) | 92 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b2aba2a4-55b3-4157-8340-f6ff29c1eb1e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b427300ea6e59b774b96a3aa248ae20c6b3b623b/articles/quickstarts/qrng.md) | 87 | 71 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/009d592b-8d6a-4360-bbcb-36d4243bf4e7_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/search.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b427300ea6e59b774b96a3aa248ae20c6b3b623b/articles/quickstarts/search.md) | 81 | 72 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/66eff262-29d7-4ea0-955f-f2ada231a2d6_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by geduardo** [comment]
Ok, I implemented both suggestions, thanks! I also fixed Grover's tutorial .md file to hide the licenses as well, because it was facing the same issue. Please, take a look to check that everything displays correctly and I think we can merge.

**Comment by cgranade** [comment]
Looks good, thanks!

**Pull request merged**